src/lib/holocene/day-of-week-picker.svelte

```diff
@@ -31,15 +31,23 @@
   <div class="flex gap-2 text-center">
     {#each genericWeekDays as { label, value }}
       {@const active = daysOfWeek.includes(value)}
-      <Button variant="secondary" {active} on:click={(e) => onClick(e, value)}>
+      <Button
+        variant="secondary"
+        class={active ? 'bg-interactive-secondary-active' : ''}
```

Comment:
refresh my memory what `active` means in this component? Is it worth adding an `active` button prop?

Reply:
I went this route for now instead of keeping the `active` button prop because this styling is for a "selected" state that's only used a few places ⬇️ 

<img width="655" alt="Screenshot 2024-05-17 at 3 48 18 PM" src="https://github.com/temporalio/ui/assets/15069288/dc8c7d45-9f2f-484c-9c3a-109789cd705c">


If we have an `active` prop we need to also consider what it would look like for the `MenuButton` component (e.g. in `event-date-filter.svelte`). I've added a design task (`DES-949`) to follow-up on this.

Reply:
Ok sounds good!